state-chain/runtime/src/lib.rs

```diff
@@ -1186,6 +1186,10 @@ type PalletMigrations = (
 		11,
 	>,
 	MigrateApicalls,
+	migrations::spec_versioned_migration::SpecVersionedMigration<
+		migrations::solana_integration::SolanaChainState,
+		151,
```

Comment:
This is not good practice generally speaking since there's no guarantee we won't bump the spec version again before the release (ie. by the time we release to mainnet the spec version might be 152). But probably ok this time - we just need to remember to change this number if the version is bumped.

Reply:
Actually I think it's safer to just remove the spec version constraint.

Reply:
will do